lib/transport_web/views/resource_view.ex

```diff
@@ -6,10 +6,25 @@ defmodule TransportWeb.ResourceView do
   def format_related_objects(nil), do: ""
   def format_related_objects(related_objects) do
     related_objects
-    |> Enum.map(fn %{"id" => id, "name" => name} -> "#{name} (#{id})" end)
-    |> Enum.join(", ")
+    |> Enum.map(fn %{"id" => id, "name" => name} -> content_tag(:li, "#{name} (#{id})") end)
```

Comment:
The pipe is no longer needed here.
It also can be rewritten with list comprehension as : 
```
for %{"id" => id, "name" => name} <- related_objects, do: content_tag(:li, "#{name} (#{id})")
```

Reply:
:+1: